src/CalculatorUI/Controls/OperatorPanelButton.cs

```diff
@@ -0,0 +1,108 @@
+            // Check PropertyMetadata arg: default, or DependencyProperty.UnsetValue
+            public static readonly DependencyProperty TextProperty =
+                DependencyProperty.Register(nameof(Text), typeof(string), typeof(OperatorPanelButton), new PropertyMetadata(default(string)));
+
+            public string Glyph
+            {
+                get { return (string)GetValue(GlyphProperty); }
+                set { SetValue(GlyphProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for Glyph.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty GlyphProperty =
+                DependencyProperty.Register(nameof(Glyph), typeof(string), typeof(OperatorPanelButton), new PropertyMetadata(default(string)));
+
+            public double GlyphFontSize
+            {
+                get { return (double)GetValue(GlyphFontSizeProperty); }
+                set { SetValue(GlyphFontSizeProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for GlyphFontSize.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty GlyphFontSizeProperty =
+                DependencyProperty.Register(nameof(GlyphFontSize), typeof(double), typeof(OperatorPanelButton), new PropertyMetadata(default(double)));
+
+            public double ChevronFontSize
+            {
+                get { return (double)GetValue(ChevronFontSizeProperty); }
+                set { SetValue(ChevronFontSizeProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for ChevronFontSize.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty ChevronFontSizeProperty =
+                DependencyProperty.Register(nameof(ChevronFontSize), typeof(double), typeof(OperatorPanelButton), new PropertyMetadata(default(double)));
+
+            public Flyout FlyoutMenu
+            {
+                get { return (Flyout)GetValue(FlyoutMenuProperty); }
+                set { SetValue(FlyoutMenuProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for FlyoutMenu.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty FlyoutMenuProperty =
+                DependencyProperty.Register(nameof(FlyoutMenu), typeof(Flyout), typeof(OperatorPanelButton), new PropertyMetadata(default(Flyout)));
+
+            protected override void OnApplyTemplate()
+            {
+                if (FlyoutMenu != null)
+                {
+                    FlyoutMenu.Closed += FlyoutClosed;
+                }
+            }
+
+            protected override void OnToggle()
+            {
+                base.OnToggle();
+
+                // CSHARP_MIGRATION: TODO:
+                // Check when IsChecked is null
```

Comment:
Looks like [ToggleButton.IsChecked Property](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.togglebutton.ischecked?view=winrt-19041) has three states as well as `property Platform::IBox<bool>^ IsChecked` regarding with below comments we can find in VS
> Return Values:
**true** if the ToggleButton is checked; **false** if the ToggleButton is unchecked; otherwise **null**. The default is **false**.

So, we should check null value before we check the value. Otherwise this C# code will throw but the original C++ won't.